lib/build.gradle

```diff
@@ -63,9 +63,10 @@ dependencies {
     implementation "com.google.dagger:dagger:2.41"
     implementation 'com.davemorrissey.labs:subsampling-scale-image-view-androidx:3.10.0'
     implementation 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
-    implementation 'com.github.MikeOrtiz:TouchImageView:3.1.1'
-    implementation "com.github.bumptech.glide:glide:4.11.0"
-    kapt "com.github.bumptech.glide:compiler:4.11.0"
+    implementation "io.coil-kt:coil:2.4.0"
+    implementation "io.coil-kt:coil-gif:2.4.0"
+    implementation "io.coil-kt:coil-svg:2.4.0"
+    implementation "io.coil-kt:coil-video:2.4.0"
```

Comment:
@mdrlzy We should define a version constant for `coin-kt` modules for reusability.